source/BankPanel.cpp

```diff
@@ -207,6 +209,15 @@ void BankPanel::Draw()
 		table.Draw(maintenance);
 		table.Advance();
 	}
+	if(assetsReturns)
+	{
+		totalPayment -= assetsReturns;
+		const auto incomeLayout = Layout(310, Truncate::BACK);
+		table.DrawCustom({"Return on ships and outfits", incomeLayout});
+		table.Advance(3);
+		table.Draw(-(assetsReturns));
+		table.Advance();
+	}
 	if(income[0] || income[1])
```

Comment:
Good idea. I combined them.

Reply:
Is there a plan to summarize this new income in the PlayerInfoPanel?

Reply:
I didn't plan to do that; it doesn't seem to fit in "salary" and it doesn't seem to fit in the "tribute" section. Do you feel it should be displayed on the PlayerInfoPanel?

Reply:
As an "income," it would feel appropriate to itemize somehow there. However, there are only two possible "items" in this list (always and only-when-unparked), so it seems like something that would warrant its own heading & itemization.

Probably it (total daily income, including salary and tribute) could be summarized as an item following "net worth"

Reply:
I thought about it a bit more and I feel that it is inappropriate to show the "operating income" information on the player screen; the Salary and Tribute items are "player-info specific", they are based on relations the player has with employers, business partners, and dominated planets. But there is a lot of other financial information that we don't show on the player screen (crew costs, operating costs, monthly mortgage payments). If a player wants to know the daily income (and the daily costs and the daily profit/loss), then the Bank-screen seems to be a much more appropriate place to go to.

If we really should add something to the player-screen, then I would suggest a "daily profit" / "daily loss" entry, where all income and all costs (including crew costs, operating costs and mortgage payments) are added up. Only displaying the income without including the costs could be misleading.